contrib/cirrus/lib.sh

```diff
@@ -139,6 +139,11 @@ die() {
     exit ${1:-1}
 }
 
+showrun() {
```

Comment:
Please remove this; it's not used, and it's confusing. (If you ever end up needing/using it in other code, then OK to add back, but please with a note in logcollector.sh indicating that you're overriding it).

Reply:
woops, good catch.